proto/cosmos/gov/v1beta2/query.proto

```diff
@@ -13,42 +13,42 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types";
 service Query {
   // Proposal queries proposal details based on ProposalID.
   rpc Proposal(QueryProposalRequest) returns (QueryProposalResponse) {
```

Comment:
Are we going to support v1beta1 queries in a separate PR?

Reply:
I wasn't sure that that had been confirmed. Marie and I had talked about it but we decided to postpone the decision. I can do this in a follow up PR if we want to

Reply:
Would we want to structure the v1beta1 stuff as a different querier entirely or append it to the v1beta2 querier

Reply:
different querier with a limited set of functionality available i think